packages/botonic-plugin-ai-agents/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "botonic-plugin-ai-agents",
+  "version": "0.36.0",
+  "main": "./lib/cjs/index.js",
+  "module": "./lib/esm/index.js",
+  "description": "Use AI Agents to generate your contents",
+  "scripts": {
+    "build": "rm -rf lib && ../../node_modules/.bin/tsc -p tsconfig.json && ../../node_modules/.bin/tsc -p tsconfig.esm.json",
+    "build:watch": "npm run build -- --watch",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "prepublishOnly": "rm -rf lib && npm i && npm run build",
+    "lint": "npm run lint_core -- --fix",
+    "lint_core": "../../node_modules/.bin/eslint_d --cache --quiet 'src/**/*.ts*'"
+  },
+  "dependencies": {
+    "@botonic/react": "^0.35.0",
```

Comment:
Plugin should not depend on botonic-react but botonic-core right?

Reply:
Yes, I totally agree, my mistake, I change it